app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```

Comment:
It's great that you're using type annotations for function arguments and return type. However, list type hinting in Python should be written with capital 'L' like `List[type]`, not `list[type]`. So, it should be `List[dict]` instead of `list[dict]`.